pages/videos/[slug].tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Layout from "../../components/layout/layout";
+import fs from "fs";
+import path from "path";
+import matter from "gray-matter";
+import ReactMarkdown from "react-markdown/with-html";
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
+import { vscDarkPlus } from "react-syntax-highlighter/dist/cjs/styles/prism";
+import Head from "next/head";
+import styles from "../../styles/video_lesson.module.scss";
+
+const CodeBlock = ({ language, value }) => {
+  return (
+    <SyntaxHighlighter language={'dart'} style={vscDarkPlus} lineProps={{style: {fontSize: '1.6rem'}}} wrapLines={true} showInlineLineNumbers={false}>
+      {value}
+    </SyntaxHighlighter>
+  );
+};
+
+export default function Lesson({ content, frontmatter }) {
+  return (
+    <Layout>
+      <Head>
+        <title>{frontmatter.title}</title>
+        <link rel="icon" href="/favicon.ico" />
+        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
+      </Head>
+      <div className={styles.content}>
+        <h1>{frontmatter.title}</h1>
+        {frontmatter.youtube ? (
+          <div className={styles.video}>
+            <iframe src={`https://www.youtube.com/embed/${frontmatter.youtube}`} />
```

Comment:
What happens if we don't give this a link for YouTube but let's say it's only a normal blog post?